clang/lib/CIR/Dialect/Transforms/LifetimeCheck.cpp

```diff
@@ -1786,6 +2173,9 @@ void LifetimeCheckPass::checkCall(CallOp callOp) {
   // part of declaration
   trackCallToCoroutine(callOp);
 
+  // Track moves via rvalue reference parameters for Value types
+  checkMoveViaCall(callOp);
```

Comment:
`checkMoveViaCall` -> `checkMoveInCallArgs`?

Reply:
Renamed to checkMoveInCallArgs (LifetimeCheck.cpp:1633), see 53869e9c70c5